instance/migrations/0053_remove_subdomain_basedomain.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-06-03 08:18
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0052_add_internal_domain_fiels'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='openedxinstance',
+            name='internal_lms_domain',
+            field=models.CharField(max_length=100, unique=True),
+        ),
+        migrations.AlterField(
+            model_name='openedxinstance',
+            name='internal_lms_preview_domain',
+            field=models.CharField(max_length=100, unique=True),
+        ),
+        migrations.AlterField(
+            model_name='openedxinstance',
+            name='internal_studio_domain',
+            field=models.CharField(max_length=100, unique=True),
+        ),
```

Comment:
@mtyaka Would it be possible to get rid of these by adding `unique=True` to the corresponding `AddField` operations in the previous migration (0052_add_internal_domain_fiels)?